src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Client/CommitmentsApiClient.cs

```diff
@@ -541,6 +541,11 @@ public Task PriorLearningDetails(long cohortId, long apprenticeshipId, PriorLear
             return _client.PostAsJson($"api/cohorts/{cohortId}/draft-apprenticeships/{apprenticeshipId}/prior-learning", request, cancellationToken);
         }
 
+        public Task PriorLearningData(long cohortId, long apprenticeshipId, PriorLearningDataRequest request, CancellationToken cancellationToken = default)
```

Comment:
Is an update to the client still required? If all requests no go via the outer api, I'd have though this wasn't needed.

Reply:
Initially I thought it was as I assumed APIM used the client. But happy to revert this

Reply:
removed